dask_geopandas/morton_distance.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import pandas as pd
+from dask_geopandas.hilbert_distance import _continuous_to_discrete_coords
+
+# Based on #https://code.activestate.com/recipes/
+# 577558-interleave-bits-aka-morton-ize-aka-z-order-curve/
```

Comment:
The original code is licensed under LGPL 2.1. Can you ensure we can use it here and under which conditions?

Reply:
If it's LGPL, we can't use it here. However, the recipe states that it's itself based on other code, which I think we can probably use if we refer directly to those.

Reply:
Also, https://github.com/trevorprater/pymorton has similar code and which is MIT, so that's fine. So in general this code (the specific bit values) seems widespread, so maybe no explicit reference is needed.